### PR TITLE
Minor changes to Packaging section

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Add description of using list of strings
as value in the OPTS variable.

Update config example accordingly

Minor formatting fixes

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -550,7 +550,7 @@ Makd supports a simple facility to make packages based on fpm_.  A simple
 wrapper program ``mkpkg`` is provided to ease the creation of scripts that use
 fpm_ to create packages.  The predefined ``pkg`` target will scan for ``*.pkg``
 files in the ``$(PKG)`` directory (by default ``$T/pkg``) and then invoke
-``mkpkg`` with them.
+``mkpkg`` with them.
```
Comment (leandro-lucarella-sociomantic) [inline README.rst]: Every time you add a trailing space a kitten dies :crying_cat_face:

File: README.rst [hunk 2]
```diff
@@ -559,9 +559,10 @@ These files are expected to be Python scripts defining two variables:
         a fpm_ command-line option. If the key is only one character (for
         example ``c``), it will be passed as ``-<key><value>`` and if it's
         more, it will be passed as ``--<key>=<value>`` (``_`` characters in the
-        key will be replaced by ``-`` for convenience). No validation is
-        performed over the keys or values, they are just passed blindly to
-        fpm_.
+        key will be replaced by ``-`` for convenience). The ``<value>`` can be a
+        string or an array of strings. In the latter case, the key is used as fpm
```
Comment (leandro-lucarella-sociomantic) [inline README.rst]: `fpm_` so it get a pretty link. Also maybe `used as the fpm_ flag` (I would drop the "cli" too, as it's the only interface fpm offers).

File: README.rst [hunk 3]
```diff
@@ -685,7 +686,8 @@ For convenience, here is a simple example:
           category = 'net',
 
           depends = FUN.autodeps(bin_path) + [ 'bash',
-            'libnew' if VAR.lsb_release == 'trusty' else 'libold' ],
+            'libnew' if VAR.lsb_release == 'trusty' else 'libold' ]
```
Comment (leandro-lucarella-sociomantic) [inline README.rst]: I prefer to keep the trailing comma, is legal in Python and it makes adding more keys to the dict in the future more hassle-free.

Comment (andreas-bok-sociomantic) [inline README.rst]: I concede,

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [inline README.rst]: What about the baby cockroaches...ok I will remove them.

**Comment by andreas-bok-sociomantic** [comment]
updated. 
new diff:

``` diff
diff --git a/README.rst b/README.rst
index 13a0603..2bb071f 100644
--- a/README.rst
+++ b/README.rst
@@ -550,7 +550,7 @@ Makd supports a simple facility to make packages based on fpm_.  A simple
 wrapper program ``mkpkg`` is provided to ease the creation of scripts that use
 fpm_ to create packages.  The predefined ``pkg`` target will scan for ``*.pkg``
 files in the ``$(PKG)`` directory (by default ``$T/pkg``) and then invoke
-``mkpkg`` with them. 
+``mkpkg`` with them.

 These files are expected to be Python scripts defining two variables:

@@ -560,9 +560,9 @@ These files are expected to be Python scripts defining two variables:
         example ``c``), it will be passed as ``-<key><value>`` and if it's
         more, it will be passed as ``--<key>=<value>`` (``_`` characters in the
         key will be replaced by ``-`` for convenience). The ``<value>`` can be a
-        string or an array of strings. In the latter case, the key is used as fpm 
-        cli flag for each item in ``<value>``. No validation is performed over the 
-        keys or values, they are just passed blindly to fpm_. 
+        string or an array of strings. In the latter case, the key is used as fpm_
+        flag for each item in ``<value>``. No validation is performed over the
+        keys or values, they are just passed blindly to fpm_.

 ``ARGS``
         a ``list()`` (array) to pass to fpm_ as positional arguments (usually the
@@ -686,7 +686,7 @@ For convenience, here is a simple example:
           category = 'net',

           depends = FUN.autodeps(bin_path) + [ 'bash',
-            'libnew' if VAR.lsb_release == 'trusty' else 'libold' ]
+            'libnew' if VAR.lsb_release == 'trusty' else 'libold' ],

         )
```

**Comment by leandro-lucarella-sociomantic** [comment]
This pull request has been rebased via `git hub pull rebase`. Original pull request HEAD was 3829582ec5e0c350bdb32535d1d5009a48e8aee8, new (rebased) HEAD is c1ccc221ae2c4b4ae351b6c30a6bd0567c33c973

**Pull request closed without merging**